content/millennium/r4/individuals/patient.md

```diff
@@ -9,4 +9,215 @@ title: Patient | R4 API
+
+    GET /Patient?:parameters
+
+_Implementation Notes_
+
+* Direct secure email will not be returned.
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true) %>
+
+### Parameters
+
+ Name                 | Required?                                    | Type       | Description
+----------------------|----------------------------------------------|------------|--------------------------------------------------------------------------
+ `_id`                | This, or any other required search parameter | [`token`]  | The logical resource id associated with the resource.
+ `identifier`         | This and/or any other search param, or `_id` | [`token`]  | A patient identifier.  Example: `urn:oid:1.1.1.1.1.1|1022228`
+ `name`               | This and/or any other search param, or `_id` | [`string`] | The start of either family or given name of the patient. Example: `John`
+ `family`             | This and/or any other search param, or `_id` | [`string`] | The start of the family name of the patient. Example: `Smith`
+ `given`              | This and/or any other search param, or `_id` | [`string`] | The start of the given name of the patient. Example: `John`
+ `birthdate`          | This and/or any other search param, or `_id` | [`date`]   | The patient's date of birth.  Example: `1990-01-01`
+ `phone`              | This and/or any other search param, or `_id` | [`token`]  | The patient's phone number. Example: `1234567891`
+ `email`              | This and/or any other search param, or `_id` | [`token`]  | The patient's email address. Example: `example@example.com`
+ `address-postalcode` | This and/or any other search param, or `_id` | [`string`] | The postal code in the address details of the patient. Example: `12345`
+ `gender`             | No                                           | [`token`]  | The gender of the patient. Example: `male`
+ [`_count`]           | No                                           | [`number`] | The maximum number of results to return. Defaults to `20`.
+
+Notes:
+
+* Either the `_id`, or a combination of `identifier` , `birthdate`, `name`, `given`, `family`, `address-postalcode`, `phone`, or `email` parameters must be provided.
+* The `gender` parameter may only be provided if at least one of `identifier` , `birthdate`, `name`, `given`, `family`, `address-postalcode`, `phone`, or `email` parameters is provided.
+* The `name`, `family`, and `given` parameters support the ':exact' modifier and will search for current names only.
+* The `identifier`, `name`, `family`, `given`, `phone`, `email`, `address-postalcode`, or `gender` parameters may be provided exactly once and may have only a single value.
+* The `birthdate` parameter may be provided twice to indicate a date range, but must contain the inclusive prefixes 'le' and 'ge'
+* The `birthdate` parameter may be provided once with the following prefixes: 'ge', 'le', 'gt', 'lt', 'eq'
+
+### Headers
+
+ <%= headers %>
```

Comment:
rebase when available to pull in this change for fhir+json https://github.com/cerner/fhir.cerner.com/commit/761f781e1eb58169b9b02d2530719b468b68a24d#diff-7718c11ed130ac00a9854e0b089f74b4L27

Reply:
Rebased this branch